dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -395,7 +400,7 @@ Path findClassLocation(String className) {
         javaClasses.add(javaClass);
       } catch (ClassNotFoundException ex) {
         // We couldn't find the class in the jar file where we found it.
-        throw new IOException("Corrupt jar file " + jar + "; could not load " + classFileName, ex);
+        corruptedClassFileNames.add(classFileName);
       } catch (ClassFormatException ex) {
         // We couldn't load the class from the jar file where we found it.
         throw new IOException(
```

Comment:
could we bubble the ClassFormatException instead? This isn't really an IO error.

Reply:
Yes, removed the catch clause.

Reply:
It turned out that this change causes #1142.